README.md

```diff
@@ -47,19 +47,17 @@ pgmonitor combines multiple open-source software packages and necessary configur
 
 ### Operating Systems
 
-- CentOS 6 or greater
-- RHEL 6 or greater
+- Prometheus/Alertmanager & Grafana: CentOS/RHEL 7 or greater
+- Exporters (node & postgres): CentOS/RHEL 6 or greater
 
 ### PostgreSQL
 
-- 9.4
-- 9.5
-- 9.6
-- 10
+- We strive to officially support all versions of PostgreSQL supported by the community itself. This means we will drop support for versions that are moved out of community support. And we will do our best to add support as soon as possible when new major versions are released.
```

Comment:
@jkatz this looks good ?

Reply:
@sharmay How does this sound:

pgmonitor supports the following major versions of PostgreSQL:

- 11, 10, 9.6, 9.5, 9.4

pgmonitor plans to support all PostgreSQL versions that are [actively supported](https://www.postgresql.org/support/versioning/) by the PostgreSQL community. Once a major version of PostgreSQL reaches its end-of-life (EOL), pgmonitor will cease supporting that major version.

Reply:
Done!

Reply:
Thank you!